lib/paper_trail/config.rb

```diff
@@ -22,9 +22,11 @@ def initialize
 
     def serialized_attributes=(value)
       if ::ActiveRecord::VERSION::MAJOR >= 5
-        warn("DEPRECATED: ActiveRecord 5.0 deprecated `serialized_attributes` "  +
+        ::ActiveSupport::Deprecation.warn(
```

Comment:
Nifty trick, I didn't realize this class existed. I like how it prints more details about the source of the warning too.